src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/src/CreateFrameworkListFile.cs

```diff
@@ -46,6 +46,8 @@ public class CreateFrameworkListFile : BuildTask
 
         public string[] TargetFilePrefixes { get; set; }
 
+        public string[] SingleFileHostIncludeList { get; set; }
```

Comment:
Improve the name of this property? (Normally want to describe the contents, not that it's a list.)

```suggestion
        public string[] SingleFileHostIncludeFilenames { get; set; }
```